notebooks/214-vision-paddle-classification/214-vision-paddle-classification.ipynb

```diff
@@ -0,0 +1,325 @@
+   "source": [
+    "from pathlib import Path\n",
+    "import os\n",
+    "import urllib.request\n",
+    "import tarfile\n",
+    "\n",
+    "mobilenet_url = \"https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/inference/MobileNetV3_large_x1_0_infer.tar\"\n",
+    "mobilenetv3_model_path = Path(\"model/MobileNetV3_large_x1_0_infer/inference.pdmodel\")\n",
+    "if mobilenetv3_model_path.is_file(): \n",
+    "    print(\"Model MobileNetV3_large_x1_0 already exists\")\n",
+    "else:\n",
+    "    # Download the model from the server, and untar it.\n",
+    "    print(\"Downloading the MobileNetV3_large_x1_0_infer model (20Mb)... May take a while...\")\n",
+    "    # create a directory \n",
+    "    os.makedirs('model')\n",
+    "    urllib.request.urlretrieve(mobilenet_url, \"model/MobileNetV3_large_x1_0_infer.tar\")\n",
+    "    print(\"Model Downloaded\")\n",
+    "\n",
+    "    file = tarfile.open(\"model/MobileNetV3_large_x1_0_infer.tar\")\n",
+    "    res = file.extractall('model')\n",
+    "    file.close()\n",
+    "    if (not res):\n",
+    "        print(\"Model Extracted to \\\"model/MobileNetV3_large_x1_0_infer\\\".\")\n",
+    "    else:\n",
+    "        print(\"Error Extracting the model. Please check the network.\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f3a72a48-494e-44f3-9ee8-b2c15c845fdc",
+   "metadata": {},
+   "source": [
+    "## Define the callback function for postprocessing"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "id": "36b52bfa-551c-4d58-a796-7a0cb7e8c357",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def callback(infer_request, i) -> None:\n",
+    "    \"\"\"\n",
+    "    Define the callback function for postprocessing\n",
+    "    \n",
+    "    :param: infer_request: the infer_request object\n",
+    "            i: the iteration of inference\n",
+    "    :retuns:\n",
+    "            None\n",
+    "    \"\"\"\n",
+    "    imagenet_classes = json.loads(open(\"utils/imagenet_class_index.json\").read())\n",
+    "    predictions = next(iter(infer_request.results.values()))\n",
+    "    indices = np.argsort(-predictions[0])\n",
+    "    if (i == 0):\n",
+    "        # Calculate the first inference time\n",
+    "        latency = time.time() - start\n",
+    "        print(\"latency:\", latency)\n",
+    "        for n in range(5):\n",
+    "            print(\n",
+    "                \"Class name:\",\"'\" + imagenet_classes[str(list(indices)[n])][1] + \"'\",\n",
+    "                \", probability:\" , predictions[0][list(indices)[n]])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c8adfba1-82d2-4526-910f-2d32530d74eb",
+   "metadata": {},
+   "source": [
+    "## Read the model file "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "id": "179ae1a7-966d-49fe-9c06-811257c57989",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from openvino.runtime import Core\n",
```

Comment:
Group imports together, put them in first cell

Reply:
In order to keep the beginner into the needs for every task with OpenVINO, I suggest @OpenVINO-dev-contest, put all imports in the first cell, but include a block of comments for those. For example, which imports are necessary for Preprocessing, which ones for Auto-Plugin, and so on.

Whit this we will avoid repeating the same imports over the notebook

Reply:
Now import has a separate cell